test-apps/presentation-test-app/src/frontend/components/tree-widget/Tree.tsx

```diff
@@ -17,7 +17,7 @@ interface Props {
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
 export const Tree: React.FC<Props> = (props: Props) => {
-  const nodeLoader = usePresentationTreeNodeLoader({
+  const { nodeLoader } = usePresentationTreeNodeLoader({

```

Comment:
I would like to make this breaking change. If we change hook to take in optional React ref than consumers will have to setup callback for `onItemsRendered` themselves. In that case I think it would be quicker for consumers to adjust code to this breaking change.

Reply:
I prefer breaking it - `@beta` still allows us to do that, especially since resolving the breaking change on consumer side is as simple as wrapping the variable name with curly braces. An important piece to this is documenting the breaking change in `NextVersion.md` - I see it's documented, but maybe not highlighted enough. Will comment on that separately.